cg/cli/utils.py

```diff
@@ -1,6 +1,13 @@
+import re
+
 import click
 
 
 def echo_lines(lines: list[str]) -> None:
     for line in lines:
         click.echo(line)
+
+
+def is_case_name_allowed(name: str) -> bool:
```

Comment:
add a couple of tests for this, one that passes and one that fails

Reply:
Added three 😎